### PR TITLE
Create comment

### PR DESCRIPTION
**Pull request by trungnn142**

### CODE REVIEW

File: app/views/comments/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="well">
+  <h4>Leave a Comment:</h4>
+  <%= form_for review.comments.build, remote: true, method: :post do |f| %>
+    <%= f.hidden_field :user_id, value: current_user.id %>
+    <%= f.hidden_field :review_id, value: review.id %>
+    <%= f.text_area :content, rows: 1, id: "comment_body_#{review.id}" %>
+    <%= f.submit "Submit", class: "btn btn-primary" %>
```
Comment (trinhducduy) [inline app/views/comments/_form.html.erb]: I18n

File: app/views/comments/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="well">
+  <h4>Leave a Comment:</h4>
```
Comment (trinhducduy) [inline app/views/comments/_form.html.erb]: i18n

File: db/seeds.rb [hunk 3]
```diff
@@ -28,13 +28,14 @@
                password_confirmation: password
 end
 
+users = User.order(:created_at)
```
Comment (trinhducduy) [inline db/seeds.rb]: remove ()

File: app/views/reviews/_review.html.erb [hunk 4]
```diff
@@ -27,11 +27,26 @@
...
           <i class="glyphicon glyphicon-comment"></i>
           <span><%= t "book.button.comment" %></span>
         <% end %>
       </div>
     </div>
+    <div class="row">
+      <div class="col-xs-11 col-xs-offset-1">
+        <section id="comments-section-<%= review.id %>">
+          <div class="comment">
+            <% current_user.comments.build %>
```
Comment (trinhducduy) [inline app/views/reviews/_review.html.erb]: Is this necessary? <% current_user.comments.build %>

File: app/controllers/comments_controller.rb [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        format.html {redirect_to :back}
+        format.js
+      end
+    else
+      flash[:danger] = t "flash.comment.error"
+    end
+  end
+
+  def destroy
+    @comment = current_user.comments.find_by id: params[:id]
```
Comment (ThuBM) [inline app/controllers/comments_controller.rb]: sử dụng detect thay find_by

File: app/views/comments/_comment.html.erb [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  <div class="media">
+    <div class="media-body">
+        <h6 class="media-heading">
+          <%= link_to comment.user.name, comment.user if comment.user_id %>
+          <small>
+            <% if current_user? comment.user %>
+              <%= link_to comment_path(comment), method: :delete,
+                remote: true, class: "pull-right" do %>
+                  <i class="glyphicon glyphicon-remove"></i>
+                <% end %>
```
Comment (ThuBM) [inline app/views/comments/_comment.html.erb]: indent

**Comment by ThuBM** [comment]
ok

**Pull request merged**